articles/quickstart/webapp/nextjs/index.yml

```diff
@@ -0,0 +1,53 @@
+title: Next.js
+beta: true
+alias:
+  - next
+  - nextjs
+  - next.js
+language:
+  - Javascript
+framework:
+  - Next.js
+author:
+  name: Rita Zerrizuela
+  email: rita.zerrizuela@auth0.com
+  community: false
+image: /media/platforms/node.png
+topics:
+  - quickstart
+contentType: tutorial
+useCase: quickstart
+seo_alias: nextjs
+show_releases: true
+show_steps: true
+requirements:
+  - Next.js 10+
+default_article: 01-login
+articles:
+  - 01-login
+snippets:
+  dependencies: server-platforms/nextjs/dependencies
+  setup: server-platforms/nextjs/setup
+  use: server-platforms/nextjs/use
+sdk:
+  name: nextjs-auth0
+  url: https://github.com/auth0/nextjs-auth0/
+github:
+  org: auth0-samples
+  repo: auth0-nextjs-samples
+  branch: master
+next_steps:
```

Comment:
Let's remove the `next_steps` section and do what we did with Express: https://github.com/auth0/docs/blob/master/articles/quickstart/webapp/express/01-login.md#whats-next

## What's next?
We put together a few examples of how to use [nextjs-auth0](https://github.com/auth0/nextjs-auth0) in more advanced use cases:

- [Protecting a Server Side Rendered (SSR) Page](https://github.com/auth0/nextjs-auth0/blob/beta/EXAMPLES.md#protecting-a-server-side-rendered-ssr-page)
- [Protecting a Client Side Rendered (CSR) Page](https://github.com/auth0/nextjs-auth0/blob/beta/EXAMPLES.md#protecting-a-client-side-rendered-csr-page)
- [Protect an API Route](https://github.com/auth0/nextjs-auth0/blob/beta/EXAMPLES.md#protect-an-api-route)
- [Access an External API from an API Route](https://github.com/auth0/nextjs-auth0/blob/beta/EXAMPLES.md#access-an-external-api-from-an-api-route)

Reply:
Changed in https://github.com/auth0/docs/pull/9557/commits/e932a625798bedc2b280ff2606c57b780ac4ef15.